src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -267,8 +263,8 @@ public static void TestDecryptSimpleTripleDes_IssuerAndSerial()
         }
 
         [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
         [OuterLoop(/* Leaks key on disk if interrupted */)]
+        [ActiveIssue(3334, PlatformID.AnyUnix)]
```

Comment:
Point taken. Deleting fallback behavior until I address how to add it to the cert before passing it to OpenSSL